### PR TITLE
Section Phrases

### LINKED ISSUE #33: Add phrase and section default seeds.

### PR DESCRIPTION
**Pull request by sagemlee**
### Description
Add section_phrases joins table, model and query tests
Change User pitch attribute to voice(string)
Add first draft of readme

### Type of change
 - [x] Bug fix (non-breaking change which fixes an issue)
 - [ ] Refactor (non-breaking change which maintains existing functionality)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

File: app/models/section_phrase.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class SectionPhrase < ApplicationRecord
```
Comment (sagemlee) [inline app/models/section_phrase.rb:1]: Was debugging for about an hour when I realized this file name was misspelled. gahhhh.

File: db/migrate/20200909035016_change_user_pitchto_user_voice_name.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+class ChangeUserPitchtoUserVoiceName < ActiveRecord::Migration[5.2]
+  def change
```
Comment (sagemlee) [inline db/migrate/20200909035016_change_user_pitchto_user_voice_name.rb:2]: I accidentally migrated this empty, that is why there are two migrations with similar names.

File: README.md [hunk 3]
```diff
@@ -1,24 +1,48 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
```
Comment (EricLarson2020) [inline README.md:4]: Thanks for adding in the readme!

File: app/graphql/types/user_type.rb [hunk 4]
```diff
@@ -6,7 +6,7 @@ class UserType < Types::BaseObject
     field :email, String, null: true
     field :password, String, null: true
     field :speed, Integer, null: true
-    field :pitch, Integer, null: true
```
Comment (EricLarson2020) [inline app/graphql/types/user_type.rb:9]: Good work changing this!

**Review by EricLarson2020 — APPROVED** [review]

File: app/models/section.rb [hunk 5]
```diff
@@ -1,5 +1,6 @@
 class Section < ApplicationRecord
   has_many :user_sections
   has_many :users, through: :user_sections
-
+  has_many :section_phrases
+  has_many :phrases, through: :section_phrases
```
Comment (cgaddis36) [inline app/models/section.rb:5]: nice migration work here!

File: app/models/section_phrase.rb (hunk 1 above)
Comment (cgaddis36) [reply app/models/section_phrase.rb:1]: gotta love it!

File: config/application.rb [hunk 6]
```diff
@@ -10,10 +10,7 @@
 require "action_mailer/railtie"
 require "action_view/railtie"
 require "action_cable/engine"
-
-#require "sprockets/railtie"
-
-
+# require "sprockets/railtie"
```
Comment (cgaddis36) [inline config/application.rb:13]: This needs to be uncommented for Heroku deploy

File: db/migrate/20200909035016_change_user_pitchto_user_voice_name.rb (hunk 2 above)
Comment (cgaddis36) [reply db/migrate/20200909035016_change_user_pitchto_user_voice_name.rb:2]: can we take one of the migrations out?

File: spec/models/section_spec.rb [hunk 7]
```diff
@@ -5,5 +5,7 @@
   describe 'relationships' do
     it {should have_many(:user_sections)}
     it {should have_many(:users).through(:user_sections)}
+    it {should have_many(:section_phrases)}
+    it {should have_many(:phrases).through(:section_phrases)}
```
Comment (cgaddis36) [inline spec/models/section_spec.rb:9]: nice testing validations here

File: spec/requests/v1/user_query_spec.rb [hunk 8]
```diff
@@ -94,6 +94,94 @@ def query(id:)
...
+          GQL
+          end
+
+    post "/v1/graphql", params: { query: query(id: bob.id)}
+    expect(response).to be_successful
+
+    reply = JSON.parse(response.body, symbolize_names: true)
+    expect(reply[:data][:user][:sections].first[:phrases]).to_not be_empty
+    expect(reply[:data][:user][:sections].first[:phrases].first[:expression]).to eq('Hello')
+    expect(reply[:data][:user][:sections].first[:phrases].last[:expression]).to eq('My name is Bob')
```
Comment (cgaddis36) [inline spec/requests/v1/user_query_spec.rb:183]: nice thorough testing here, Great work!!

**Pull request closed without merging**